projects/kit/components/slider/slider.style.less

```
@@ -75,37 +75,37 @@
 
         &:hover,
         :active& {
-            transform: var(--tui-slider-thumb-transform, @transform-noop) scale(@active-thumb-scale-factor[@@size]);
+            transform: var(--tui-slider-thumb-transform, @transform-noop) scale(@active-thumb-scale-factor);
         }
     }
 
-    :focus-visible& {
+    :focus-visible & {
```

Why ?

---

Without size property we does not have so much logic-branching.
Let's change code to the such way to fix stylelint error (`stylelint-disable` code comment does not suppress error)
```
➜ stylelint projects/kit/components/slider/slider.style.less --config package.json                                                                                                 

projects/kit/components/slider/slider.style.less
  82:5  ✖  Cannot parse selector (Error: Misplaced parenthesis.)  parseError
  82:5  ✖  Cannot parse selector (Error: Misplaced parenthesis.)  parseError

✖ 2 problems (2 errors, 0 warnings)
```

```less
:host:focus-visible {
    @outline: 0 0 0 2px inset var(--tui-border-focus);

    &::-webkit-slider-thumb {
        box-shadow: @outline;
    }

    &::-moz-range-thumb {
        box-shadow: @outline;
    }
}
```

---

sure, great idea! fixed